README.md

```diff
@@ -37,6 +37,9 @@ sh tests/clean.sh
 
 Sudo permission is required.
 
+## MSRV
+The current MSRV(minimum supported rust version) is 1.60.
```

Comment:
I don't think a 0.1 crate needs to specify/hold itself to some specific MSRV or even specify it 😇. Specifically, I'm not sure if anyone in future changes will take the care to update it.